crates/wash-lib/src/cli/registry.rs

```diff
@@ -99,4 +99,8 @@ pub struct RegistryPushCommand {
 
     #[clap(flatten)]
     pub opts: AuthOpts,
+
+    /// Push the artifact monolithically instead of chunked
+    #[clap(long = "monolithic-push", env = "WASH_MONOLITHIC_PUSH")]
```

Comment:
I'm happy to word this differently if anyone has suggestions on alternative naming for this option, I went with what felt obvious, but it doesn't necessarily convey the intent in the best possible way (not that anyone should have to know the difference really).